src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -108,6 +109,30 @@ public static JsonObject createStaffSlipContext(Request request) {
     return createStaffSlipContext(request.getItem(), request);
   }
 
+  public static JsonObject addPrimaryServicePointNameToStaffSlipContext(JsonObject entries, ServicePoint primaryServicePoint) {
+    log.info("addPrimaryServicePointNameToStaffSlipContext:: parameters entries: {}", entries);
```

Comment:
The log should be set to debug level and include the second parameter in its message